src/redux/request.js

```diff
@@ -241,7 +241,8 @@ export function createRequestEpic({
     action$.ofType(REQUEST_MAKE)
       .filter((action) => action.name === name)
       .switchMap(({ data = {}, options = {} }) => {
-        let { okapi } = getState();
+        let state = getState();
+        let { okapi } = state;
```

Comment:
This is unnecessary as well now. You can just ignore the changes in this whole file